### PR TITLE
Allow optionally-defined Assembly for LethalConfigManager#AddConfigItem

### PR DESCRIPTION
**Pull request by FerusGrim**

### CODE REVIEW

File: Assets/Scripts/LethalConfigManager.cs [hunk 1]
```diff
@@ -66,9 +66,9 @@ internal static void AutoGenerateMissingConfigsIfNeeded()
         /// <summary>
         ///     Adds a config item to the calling's mod entry UI.
         /// </summary>
-        public static void AddConfigItem(BaseConfigItem configItem)
+        public static void AddConfigItem(BaseConfigItem configItem, Assembly assembly = null))
```
Comment (AinaVT) [inline Assets/Scripts/LethalConfigManager.cs:69]: There's two closing parenthesis, syntax error.

Comment (FerusGrim) [reply Assets/Scripts/LethalConfigManager.cs]: Resolved

**Comment by AinaVT** [comment]
This changes the signature of the method, which makes mods using older versions to break. Probably will have to create a new method altogether, even if using the same name.

**Review by AinaVT — APPROVED** [review]
Looks good!

**Pull request merged**